.github/actions/ci-setup/action.yml

```diff
@@ -11,5 +11,5 @@ runs:
         cache: "yarn"
 
     - name: Install Dependencies
-      run: yarn install --frozen-lockfile --ignore-scripts
```

Comment:
I had to remove this to execute `patch-package`

Reply:
I don't know what the implications are @andrasbacsai